coops-ui/tests/unit/Directors.spec.ts

```diff
@@ -165,20 +165,23 @@ describe('Directors as a COOP', () => {
     expect(directorListUI[1].innerHTML).toContain('<span>Cease</span>')
   })
 
-  it('disables buttons/actions when instructed by parent component', () => {
-    // invalidate AGM Date
-    const wrapper = createWrapper(vm)
-    wrapper.setProps({ componentEnabled: false })
-    // confirm that flag is set correctly
-    expect(vm.componentEnabled).toEqual(false)
+  it('disables buttons/actions when instructed by parent component', done => {
+    // clear enabled prop
+    vm.componentEnabled = false
```

Comment:
I don't know if this has any real value but if we weren't mounting the Component in the `beforeEach`  ( which we are in this test suite ) we could pass the propsData in when we mount the component to test the local property a little more organically. ie `
    const Constructor = Vue.extend(Directors)
    const instance = new Constructor({ store, vuetify, propsData: { componentEnabled: false } })
    vm = instance.$mount()`

No change at all required here, just curious if think this approach would cover more in the unit test opposed to setting it explicitly?

Reply:
Good idea... for future refactoring :)